db/seeds.rb

```diff
@@ -1,7 +1,12 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+bridge = Bridge.create(name: 'My First Bridge', payload: '', inboundURL: 'https://bridgeapi.dev/b1234/inbound', outboundURL: 'https://wowservice.io/new/23847923864', method: 'post', retries: 5, delay: 15)
+
+bridge.env_vars << EnvVar.create(key: 'database', value: 'a102345ij2')
+bridge.env_vars << EnvVar.create(key: 'database_password', value: 'supersecretpasswordwow')
+bridge.headers << Header.create(key: 'X_API_KEY', value: 'ooosecrets')
+bridge.headers << Header.create(key: 'Authentication', value: 'Bearer 1oij2oubviu3498')
```

Comment:
Does this actually add a header to the bridge? I know this will add it to the objects array but does the headers `bridge_id` column get set by this?

Reply:
It does!